docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Write Lightning Web Components in VS Code
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
+- [Code Completion](#code-completion)
+- [View Documentation on Hover](#view-documentation-on-hover)
+- [View Errors Messages on Hover](#view-error-messages-on-hover)
+- [Linting](#linting)
+
+## Code Completion
+
+The Lightning Web Components Extension builds on VS Code's code completion capabilities by completing resource names in HTML and JavaScript files.
+
+VS Code completes static resources. In this example, it completes the `chartjs` object in the `libChartjs` component. `libChartjs` refers to the third-party JavaScript library [chartjs](https://www.chartjs.org/). Include it in your code by importing `loadScript` from the Lightning Web Components `platformResourceLoader` module. Then, add the statement `import chartjs from '@salesforce/resourceUrl/chart';`. Here, VS Code fills in the resource's name when we add it in the import statement.
+
+![Static Resource Completion](./images/vscode_lwc_staticresource.png)
+
+VS Code also completes Lightning web components imports from the `lwc` module. Here, it completes `LightningElement`. When hovered over, `LightningElement` displays information about the `LightningElement` class. 
+
+![LWC Module Completion](./images/vscode_lwc_js.png)
+
+Code completion is also provided for HTML tags and attributes for components in the `lightning` and `c` namespaces. Here, with `c-view-source`, VS Code completes its source attribute.
+
+![HTML Attribute Completion](./images/vscode_lwc_html_attr.png)
+
+To view the source of a `c` namespace component, press **command** + **click**. 
```

Comment:
The example image here doesn't really make sense with the text

Reply:
which example, the view source image is the result of what happens when you press command + click. I don't know how much the image really adds to this text, I can remove. what is the equivalent command on windows?

Reply:
CMD+Click actually navigates you to the component itself. Holding CMD while hovering over the component with your cursor is what will show you the source file in a hover. This is probably better as an animated gif but the picture you currently have doesn't show either thing I mentioned